apps/salesforce-commerce-cloud/src/locations/ConfigScreen.tsx

```diff
@@ -156,7 +171,7 @@ const ConfigScreen = () => {
           <FormInput
             key={input.id}
             input={input}
-            value={parameters[input.id as keyof AppInstallationParameters]}
+            value={parameters[input.id as keyof AppInstallationParameters]!}
```

Comment:
is there perhaps a better fallback option here as opposed to enforcing it using `!` ? There may not be, but curious around the context here